src/lastRevision.bat

```diff
@@ -16,8 +16,9 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 
 rem Find the current revision, store it in a file, for DOS
-svn info | findstr Revision > %1
+git rev-parse HEAD > %1
 
-For /F "tokens=1,2 delims= " %%a In (%1) Do (
-	echo lastRevision=%%b> %1
+For /F "tokens=* delims= " %%a In (%1) Do (
```

Comment:
Thanks @arshadmohammad!

I was hoping for some committer/contributor with access to a Windows box to help me out with the BAT script. Did you test it?